Solutions/GoogleWorkspaceReports/Hunting Queries/GWorkspaceDocumentSharedPublicily.yaml

```diff
@@ -16,6 +16,7 @@ relevantTechniques:
 query: |
   GWorkspaceActivityReports
   | where TimeGenerated > ago(24h)
+  | where isnotempty(Visibility)

```

Comment:
isnotempty check is not needed as we are comparing in the next line for the same field.

Reply:
fixed